nemo/collections/asr/models/clustering_diarizer.py

```diff
@@ -442,19 +452,18 @@ def diarize(self, paths2audio_files: List[str] = None, batch_size: int = 0):
             AUDIO_RTTM_MAP=self.AUDIO_RTTM_MAP,
             out_rttm_dir=out_rttm_dir,
             clustering_params=self._cluster_params,
+            device=self._speaker_model.device,
         )
-
-        # Scoring
-        score = score_labels(
-            self.AUDIO_RTTM_MAP,
-            all_reference,
-            all_hypothesis,
-            collar=self._diarizer_params.collar,
-            ignore_overlap=self._diarizer_params.ignore_overlap,
-        )
-
-        logging.info("Outputs are saved in {} directory".format(os.path.abspath(self._diarizer_params.out_dir)))
-        return score
+        self.log_info("Outputs are saved in {} directory".format(os.path.abspath(self._diarizer_params.out_dir)))
```

Comment:
Is this necessary change? If RTTM is not provided, it does not evaluate and skip.

Also, I would prefer putting return to `else` case although it does not change anything.

Reply:
ah you are right, this does nothing! reverted this.